### DISCUSSION TITLE
Select2_from_ajax_grouped

### DISCUSSION THREAD

**Discussion by aberaldo** [Laravel-Backpack/community-forum#858] [General] [answered]
I need a grouped selector (like select2_grouped) with values loaded from ajax. Is there a way to do this?

**Comment by jcastroa87** [comment, marked as answer]
Hello @aberaldo 

We do have not such a field on Backpack right now, but I think you can achieve this goal by copy select2_from_ajax

```
php artisan backpack:field select2_from_ajax_grouped --from=select2_from_ajax
```

Then you can make the customizations needed to create a new field.

Cheers.

**Reply by aberaldo** [reply]
Hello @jcastroa87 

Thanks for the response. I will try that. But its a good issue to be included in a future release. 
If ive a solution i will share with you :-)

Cheers

**Comment by jcastroa87** [comment]
Thank you for contacting us, I hope that with these steps you can achieve your goal and if you need our help again you can reopen the issue or create a new one if necessary.

CHeers.
